source/index.html.haml

```diff
@@ -5,34 +5,35 @@ title: Hey, I'm Amber Aultman
 %header
     %nav.home
     %section.intro
-        %h1 
-            Hey, 
+        %h1
+            Hey,
             %br
             I'm Amber.
-        %p{:onclick => "showHide()"} 
+        %button{:onclick => "showHide()"}
```

Comment:
Here, I converted the the `p` element to a `button` to be a little more accessible for keyboard/screen reader users. This way you can toggle the dropdown via tabbing through elements and hitting return/enter when you reach the button.

Reply:
So smart, thank you!!